docs/reference-manual/common-practices/2-switches.md

```diff
@@ -0,0 +1,95 @@
+
+![Consecutive switch isolated](/score-docs/assets/images/reference-manual/common-practices/seqIsolate.gif "Consecutive switch isolated")
+
+In context it may be preferable to switch at evry press of the button, instead of having to hold for the first interval and let go for the second. Adding a minimum duration on each of the toggled intervals ensures that the value of the button can go back to false without validating the next trigger. This method also protects the switch from accidental "double taps".
+
+![Tap once to switch](/score-docs/assets/images/reference-manual/common-practices/tapToSwitch.gif "Tap once to switch")
+
+## Parallel switches
+
+Consecutive switches shawn above can sequence an arbitrary number of intervals, but only in the same order every time. To enable switching as well as reordering, we can rely on conditional branches. Let's first create another device for control. For this example we will add a [Mapper Device](/score-docs/docs/references/devices-types/mapper-device.html) by simply copying and pasting the script below. We'll name it "Structure".
+{% highlight qml %}
+import Ossia 1.0 as Ossia
+
+Ossia.Mapper
+{
+    function createTree() {
+        return [
+        {
+            name: "Branch",
+            type: Ossia.Type.Int,
+            write: function(v) { return v.value; }
+        }            
+        ];
+    }
+}
+{% endhighlight %}
+
+![Structure mapper](/score-docs/assets/images/reference-manual/common-practices/structureMapper.gif "Structure mpper")
+
+Parallel branches start from the same `Sync`. Syncing them at the end too will also make them responsive to the same trigger and allow us to only set up one transition to loop the entire structure.
+
+![Parallel sync](/score-docs/assets/images/reference-manual/common-practices/parallelSync.gif "Parallel sync")
+
+To add conditions for each start state of parallel intervals, they have to be splitted into individual events with the "Split condition" fonction. The state connecting the transition back to the initial `Sync` does not require a condition. Finally, a trigger on the end `Sync` can reset the loop with every new value set in the mapper.
+
+![Conditions](/score-docs/assets/images/reference-manual/common-practices/conditions.gif "Conditions")
+
+It is now possible to switch between parallel intervals when setting a value to the "Branch" parameter. To control the start of the structure when played the first time, we have to avoid adding a trigger to the start sync of the loop. It would add an idol state as we experienced before and skip the evaluation of every other value. Instead we can isolate this switch from it's initial trigger with another transition, similar to our solution to [looping parallel branches](/score-docs/docs/reference-manual/common-practices/1-looping.html#loop-branches).
```

Comment:
idol => idle
 it's initial trigger => its initial trigger

Reply:
done with 4219439